axis_frontend/src/components/Header.tsx

```
@@ -0,0 +1,49 @@
+          </div>
+          <div>
+            <h1 className="text-lg font-semibold text-gray-900 dark:text-white">
+              Alchemy
+            </h1>
+            <p className="text-xs text-gray-500 dark:text-gray-400">
+              Enterprise Platform
+            </p>
+          </div>
+        </div>
+        <nav className="flex items-center gap-1">
+          <button className="px-3 py-2 text-sm text-gray-700 dark:text-gray-300 hover:bg-gray-100 dark:hover:bg-gray-800 rounded-md transition-colors">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)